spec.html

```diff
@@ -34258,8 +34258,12 @@ <h1>String.prototype.matchAll ( _regexp_ )</h1>
           1. Let _rx_ be ? RegExpCreate(_regexp_, *"g"*).
           1. Return ? Invoke(_rx_, @@matchAll, « _S_ »).
         </emu-alg>
-        <emu-note>This method is intentionally generic, it does not require that its *this* value be a String object. Therefore, it can be transferred to other kinds of objects for use as a method.</emu-note>
-        <emu-note>Similarly to `String.prototype.split`, `String.prototype.matchAll` is designed to typically act without mutating its inputs.</emu-note>
+        <emu-note>
+          <p>This method is intentionally generic; it does not require that its *this* value be a String object. Therefore, it can be transferred to other kinds of objects for use as a method.</p>
+        </emu-note>
+        <emu-note>
+          <p>Similarly to `String.prototype.split`, `String.prototype.matchAll` is designed to typically act without mutating its inputs.</p>
+        </emu-note>
```

Comment:
let's revert all the lines that aren't changing any words, only changing indentation and whitespace

Reply:
I'm sorry I didn't get it. What exactly do I need to do.
Other place has a `p` tag embedded in it, so I added it here.

Reply:
There's no need for that, I think.

Reply:
(An `<emu-note>` with a single paragraph of content usually puts it in a `<p>` element: ~550 have it vs ~20 don't.)

Reply:
ah, fair enough